vyper/ast/utils.py

```diff
@@ -15,13 +17,14 @@
 from vyper.parser.parser_utils import (
     annotate_ast,
 )
-from vyper.parser.pre_parser import (
-    pre_parse,
-)
 from vyper.utils import (
     iterable_cast,
 )
 
+from . import (
+    nodes as vy_ast,
+)
```

Comment:
Just `import .nodes as vy_ast` directly. Add a `noqa` if flake8 is giving you problems.